unison-cli/src/Unison/Codebase/Editor/Input.hs

```diff
@@ -109,7 +109,7 @@ data Input
   | -- Move = Rename; It's an HQSplit' not an HQSplit', meaning the arg has to have a name.
     MoveTermI Path.HQSplit' Path.Split'
   | MoveTypeI Path.HQSplit' Path.Split'
-  | MoveBranchI (Maybe Path.Split') Path.Split'
+  | MoveBranchI Path.Path' Path.Path'
```

Comment:
We can now move from anywhere to anywhere, including the root (with confirmation)